### DISCUSSION TITLE
Change log level at runtime

### DISCUSSION THREAD

**Discussion by yadaom** [quarkusio/quarkus#22227] [Q&A]
Hello Everyone,

I am not able to figure out how to change the log level in Quarkus at runtime when my container is running.
For example, my default values in my `application.properties` are:
```
quarkus.log.level=INFO
quarkus.log.category."com.stock.bid.user".level=INFO
```
Once my application is up and running, I want to enable DEBUG level logs just for a specific package.
Like:
`quarkus.log.category."com.stock.bid.user".level=DEBUG`

I am not able to figure how todo this in a running application.

Please advice.

**Comment by maxandersen** [comment]
We do not have that available by default for security reasons but you can use something like https://github.com/quarkiverse/quarkus-logging-manager which provides a way to do so. 

Please read and follow the steps mentioned in the security section of that extension.
